Gemfile

```diff
@@ -5,10 +5,10 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
+gem 'rails', '4.0.12'
```

Comment:
/me trembles. I don't see many other reasonable options, let's try to merge this one as early as possible after 1.8 so that we don't merge 4.1 right before 1.9